### ISSUE TITLE
Add aria-describedby to improve image/caption relationship

### ISSUE THREAD

**Issue by karmatosed** [WordPress/gutenberg#1711]
Noting this should be thought of in output in editor, maybe.

Ticket on trac: https://core.trac.wordpress.org/ticket/34595

**Comment by mtias** [comment]
Going to close as this should happen outside the editor.
